src/services/ethereum/transaction.ts

```diff
@@ -189,6 +193,14 @@ export class TransactionExecutor implements ITransactionExecutor {
     return { res: true };
   }
 
+  private async markAllTransactions(scheduled: IScheduled, transactonList: IScheduled[]) {
```

Comment:
`markAllTransactions` is too generic of a name - mark as what? Change to e.g. `markTransactionsStale`